resources/testharness.js

```diff
@@ -3963,7 +4060,7 @@
 
     Output.prototype.show_status = function() {
         if (this.phase < this.STARTED) {
-            this.init();
+            this.init({});
```

Comment:
What's this?

Reply:
`Output.prototype.init` expects a `properties` argument that it unconditionally checks--

.... maybe we should instead be ensuring that the output object gets manually initialized, but this line raises an unconditional error without the change